### ISSUE TITLE
`package.json` without a `main` section does not get shipped to the client

### ISSUE THREAD

**Issue by tmeasday** [meteor/meteor#6582]
```
meteor create foo --release 1.3-rc.8
meteor npm install react-mounter
```

In a client file:
```js
require('react-mounter/package.json');
```

**Comment by benjamn** [comment]
Related: https://github.com/tmeasday/check-npm-versions/issues/2

**Comment by benjamn** [comment]
This works for me on `release-1.3` (58a7082aa30fd97475c436bcdae8c41177cf7294).

**Comment by benjamn** [comment]
Maybe the `require` has to be in a package file?

**Comment by tmeasday** [comment]
Ah, sorry, bad repro. Try:

```
require('react-mounter');
require('react-mounter' + '/package.json')
```

**Comment by tmeasday** [comment]
Confirmed fixed by https://github.com/meteor/meteor/commit/dc0345b0ff4dc7809c3742145bf6d49eb47ffa27

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.10`](https://github.com/meteor/meteor/issues/6266#issuecomment-200906878).
